artifactory/commands/gradle/resources/jfrog.init.gradle

```diff
@@ -0,0 +1,31 @@
+def artifactoryUrl = '{{ .ArtifactoryURL }}'
+def artifactoryRepoKey = '{{ .ArtifactoryRepositoryKey }}'
+def artifactoryUsername = '{{ .ArtifactoryUsername }}'
+def artifactoryAccessToken = '{{ .ArtifactoryAccessToken }}'
+
+gradle.settingsEvaluated { settings ->
+    settings.pluginManagement {
+        repositories {
+            maven {
+                url "${artifactoryUrl}/${artifactoryRepoKey}"
```

Comment:
lets validate that artifactoryUrl never contains a slash at the end. lets trimprifix.
lets change artifactoryRepoKey to gradleRepoName